opentutor_classifier_api/pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool]
+[tool.poetry]
+name = "opentutor_classifier_api"
+version = "1.0.0"
+description="opentutor_classifier_api"
+authors=[]
+
+[tool.poetry.dependencies]
+python = "~3.8.4"
+celery = "==5.0.5"
```

Comment:
looks to me like all the versions are still locked, and the idea is to see whether we can get them to use semver...

so as just one example...could this not be

```poetry
celery = "^5.0.5"
```

...which would mean "can be updated to any version between 5.05 and <6"

...and if we use the carot versions and then run `make update-deps` (`poetry update`)...
  - does the app still pass tests?
  - what do the changes look like in poetry lock file (this is more for us just gaining better under-the hood undetstanding)